package.json

```diff
@@ -127,11 +127,10 @@
     "sass": "^1.43.4",
     "sass-loader": "^12.3.0",
     "stylelint": "^15.11.0",
-    "stylelint-config-prettier": "^9.0.4",
```

Comment:
Not needed anymore: https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules